src/providers/ThemeProvider.tsx

```diff
@@ -13,19 +14,22 @@ interface Context {
 }
 const [useCtx, Provider] = createCtx<Context>();
 
-export const defaultThemeType: ThemeType = ThemeType.LIGHT;
+// export const defaultThemeType: ThemeType = ThemeType.LIGHT;
 
 interface Props {
   children?: React.ReactElement;
   // using initial ThemeType is essential while testing apps with consistent ThemeType
-  initialThemeType?: ThemeType;
+  // initialThemeType?: ThemeType;
```

Comment:
Removing this will make generate more work for you since some of the tests will break. What if just setting `initialThemeType` directly above?